CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * (types/coin) [#14715](https://github.com/cosmos/cosmos-sdk/pull/14715) `sdk.Coins.Add` now returns an empty set of coins `sdk.Coins{}` if both coins set are empty.
     * This is a behavior change, as previously `sdk.Coins.Add` would return `nil` in this case.
+* (reflection) [#14838](https://github.com/cosmos/cosmos-sdk/pull/14838) We now require that all proto files' import path (i.e. the OS path) matches their fully-qualified package name. For example, proto files with package name `cosmos.my.pkg.v1` should live in the folder `cosmos/my/pkg/v1/*.proto`.
```

Comment:
I added a changelog entry. @julienrbrt I think this needs an UPGRADING.md entry TBH, or else if app devs don't do this, they'll see an error, and it's not so obvious. I can work on that in a follow-up.

Reply:
Makes sense 👍🏾